tests/h/session_test.py

```diff
@@ -12,223 +12,213 @@ def __init__(self, pubid, name, is_public=False):
         self.is_public = is_public
 
 
-def test_model_sorts_groups(authenticated_request):
-    authenticated_request.set_groups([
-        FakeGroup('c', 'Group A'),
-        FakeGroup('b', 'Group B'),
-        FakeGroup('a', 'Group B'),
-    ])
-    session_model = session.model(authenticated_request)
+class TestModel(object):
+    def test_sorts_groups(self, authenticated_request):
+        authenticated_request.set_groups([
+            FakeGroup('c', 'Group A'),
+            FakeGroup('b', 'Group B'),
+            FakeGroup('a', 'Group B'),
+        ])
+        session_model = session.model(authenticated_request)
 
-    ids = [group['id'] for group in session_model['groups']]
+        ids = [group['id'] for group in session_model['groups']]
 
-    assert ids == ['__world__', 'c', 'a', 'b']
+        assert ids == ['__world__', 'c', 'a', 'b']
 
+    def test_world_group_is_included(self, authenticated_request):
```

Comment:
What was the reason for changing this from `is_public` to `is_included`? Because the test still checks whether the group is public.